### ISSUE TITLE
Support file missing/invalid bug

### ISSUE THREAD

**Issue by richpjames** [jlengstorf/netlify-plugin-visual-diff#3]
I'm getting this message when trying to deploy. 

Full log is here but it's at step 6. postBuild: https://app.netlify.com/sites/monitor-books/deploys/5f0076188f1c77000769cb31

Let me know if you need any further info

**Comment by giltayar** [comment]
@richpjames Am I missing something? I can definitely see the plugin executing, but I don't see any mention of it in the `package.json` or `netlify.toml`. Also, the latest version of the visual diff plugin is `1.2`, and this is running `1.1.3`. @jlengstorf

**Comment by jlengstorf** [comment]
looks like this was installed via the Netlify UI. I just opened a pull request to bump the package version: https://github.com/netlify/plugins/pull/107

**Comment by richpjames** [comment]
Yeah it was installed via UI hence no trace of it in the `package.json` or `netlify.toml`

**Comment by jlengstorf** [comment]
@richpjames if you want to test the latest version to see if you still get this issue, you can install it manually at `@latest`

otherwise, the version bump will be rolled out soon and (hopefully) this will be resolved
